doc/Oauth2-Configuration.md

```diff
@@ -0,0 +1,32 @@
+# Oauth2 Configuration
+You can use any IDP which supports Oauth2 like [authentik](https://goauthentik.io) for authentification with LibreBooking
```

Comment:
nit: would be nice to define `IdP` for people.

Reply:
i added the full name and hope that makes it more clear.